controller/main_controller.php

```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```

Comment:
I finally made friends TortoiseHg and Git (sorry for the couple commits trash, checking the work), so I can do normal commit.
I can remove these lines in the project, if no one wants to do it.